### DISCUSSION TITLE
Hover Dropdown inside normal dropdown

### DISCUSSION THREAD

**Discussion by ritvij14** [saadeghi/daisyui#2117] [Q&A] [answered]
So I have this kind of component:

```javascript
  return (
    <div className="dropdown dropdown-bottom font-beVietnamPro">
      <label
        tabIndex={0}
        className="btn-ghost btn flex bg-white text-sm font-medium normal-case tracking-[-0.41px] text-[#3F2BCB] shadow-[0_0_16px_0_rgba(0,0,0,0.08)] hover:bg-slate-100"
      >
        <p className="mr-3">Text 1</p>
        <div className="divider divider-horizontal mx-0 px-0 "></div>
        {DownArrow}
      </label>
      <ul
        tabIndex={0}
        className="dropdown-content menu z-[1] w-full rounded-lg bg-white p-2 shadow-[0_0_32px_0_rgba(0,0,0,0.12)]"
      >
        <li className="dropdown dropdown-open">
          <label
            tabIndex={0}
            className="flex justify-between rounded-lg bg-white p-2 text-sm font-normal normal-case tracking-[-0.41px] text-[#201B3F] shadow-[0_0_16px_0_rgba(0,0,0,0.08)] hover:bg-slate-100"
          >
            <div className="flex">{ExportIcon}&nbsp;&nbsp;Text2</div>
            {ChevronRight}
          </label>
          <ul
            tabIndex={0}
            className="dropdown-content menu z-[2] ml-3 mt-1 w-full -translate-y-full rounded-lg bg-white p-0 shadow-[0_0_32px_0_rgba(0,0,0,0.12)]"
          >
            <li>
              <a className="text-sm font-normal tracking-[-0.41px] text-[#201B3F] focus:bg-slate-100">
                Text 3
              </a>
            </li>
          </ul>
        </li>
      </ul>
    </div>
  );
```

And this is how it looks in action:

https://github.com/saadeghi/daisyui/assets/51456744/ef6d0881-ab8d-456e-b0ee-923082f41c61

As can be seen, the second dropdown closes as soon as the hover is gone, even though it has the `dropdown-open` class in the code above. So is there a way around this behavior? What I want is the second dropdown to **open only on clicking the first's option and then stay open**

**Comment by saadeghi** [comment]
There is no style for nested dropdowns (a dropdown inside another dropdown) and the reason is, it would result poor accessibility. Dropdown works with CSS `:focus` and having 2 dropdown inside eachother would create issues for keyboard navigation.  
Instead of that, you may want to use a submenu like this: https://play.tailwindcss.com/ok6siN3jxF

**Reply by hirtie-maxim** [reply]
@saadeghi so what would be the solution for it if I need one? Just to create a custom solution w/o using daisy ui? At least would I be able to reuse the styles?

**Reply by saadeghi** [reply]
@hirtie-maxim What is the exact behavior your'e looking for?  
Because If you want a dropdown to open with focus and close with losing focus, it doesn't make sense to nest dropdowns inside each other... Only one element on the page can be focused at a time so as soon as the child dropdown gets the focus, the parent dropdown is expected to close.

**Comment by ritvij14** [comment]
@saadeghi hello again. Regarding this, we wanted to give this solution a try as well but from the looks, why doesn't the arrow look like yours? (we want the chevron style and on the right side) And also the divider on the left.

Trying this from the daisy UI Examples:

<img width="224" alt="image" src="https://github.com/saadeghi/daisyui/assets/51456744/c3790713-774f-4ffc-be79-f983ab18aede">

```typescript
<details className="dropdown mb-32">
      <summary className="btn m-1">open or close</summary>
      <ul className="dropdown-content menu rounded-box z-[1] w-52 bg-base-100 p-2 shadow">
        <li>
          <a>Item 1</a>
        </li>
        <li>
          <details open>
            <summary>Parent</summary>
            <ul>
              <li>
                <a>level 2 item 1</a>
              </li>
              <li>
                <a>level 2 item 2</a>
              </li>
              <li>
                <details open>
                  <summary>Parent</summary>
                  <ul>
                    <li>
                      <a>level 3 item 1</a>
                    </li>
                    <li>
                      <a>level 3 item 2</a>
                    </li>
                  </ul>
                </details>
              </li>
            </ul>
          </details>
        </li>
        <li>
          <a>Item 2</a>
        </li>
      </ul>
    </details>
```

**Reply by saadeghi** [reply, marked as answer]
Are you using the latest version of daisyUI?

**Reply by ritvij14** [reply]
@saadeghi thanks, yea that was it
